owncloudApp/src/main/java/com/owncloud/android/presentation/transfers/TransferListComposeFragment.kt

```diff
@@ -0,0 +1,532 @@
+                    LinearProgressIndicator(
+                        modifier = Modifier
+                            .layoutId("upload_progress_bar")
+                            .fillMaxWidth()
+                            .padding(top = 8.dp, bottom = 8.dp),
+                        color = colorResource(id = R.color.color_accent),
+                        backgroundColor = colorResource(id = R.color.filelist_icon_background),
+                        progress = checkProgress(transfer, workInfo) / 100f
+                    )
+                }
+
+                Text(
+                    modifier = Modifier
+                        .layoutId("upload_account"),
+                    color = colorResource(id = R.color.list_item_lastmod_and_filesize_text),
+                    maxLines = 1,
+                    fontSize = 12.sp,
+                    text = checkAccount(transfer)
+                )
+                if (space != null) {
+                    SpacePathLine(space = space, parentPath = path)
+                }
+            }
+            Box(
+                modifier = Modifier
+                    .width(56.dp)
+                    .height(72.dp)
+                    .padding(horizontal = 8.dp),
+                contentAlignment = Alignment.Center
+            )
+            {
+                if (transfer.status != TransferStatus.TRANSFER_SUCCEEDED) {
+                    val imageResource: Int = when (transfer.status) {
+                        TransferStatus.TRANSFER_IN_PROGRESS, TransferStatus.TRANSFER_QUEUED -> {
+                            R.drawable.ic_action_cancel_grey
+                        }
+
+                        TransferStatus.TRANSFER_FAILED -> {
+                            R.drawable.ic_action_delete_grey
+                        }
+
+                        else -> {
+                            R.drawable.ic_action_delete_grey
+                        }
+                    }
+                    IconButton(onClick = { transfersViewModel.cancelUpload(transfer) }) {
+                        Icon(
+                            modifier = Modifier
+                                .width(25.dp)
+                                .height(25.dp),
+                            painter = painterResource(id = imageResource),
+                            tint = colorResource(id = R.color.half_black),
+                            contentDescription = "Button"
+                        )
+                    }
+                }
+            }
+        }
+        Divider(
+            modifier = Modifier
+                .fillMaxWidth(),
+            color = colorResource(id = R.color.filelist_icon_background),
+            thickness = 1.dp
+        )
+    }
+
+    @Composable
+    fun SpacePathLine(space: OCSpace, parentPath: String) {
+        val spaceName: String
+        val spaceImage: Int
+
+        if (space.isPersonal) {
+            spaceName = "Personal"
```

Comment:
There is already a String resource for this

Reply:
Done ✅